neoscore/core/path.py

```diff
@@ -478,19 +479,23 @@ def move_to(self, x: Unit, y: Unit, parent: Optional[PositionedObject] = None):
             parent: An optional parent, whose position the target coordinate will
                 be relative to.
         """
+        self._current_subpath_start = (Point(x, y), parent or self)
         self.elements.append(MoveTo(Point(x, y), parent or self))
 
     def close_subpath(self):
         """Close the current sub-path and start a new one at the local origin.
 
-        This is equivalent to ``move_to(Unit(0), Unit(0))``
-
-        Note:
-            This convenience method does not support point parentage.
-            If you need to anchor the new point, use an explicit
-            ``move_to(Unit(0), Unit(0), parent)`` instead.
+        This is equivalent to ``line_to(Unit(0), Unit(0))``
+        drawing a line back to the local origin
+        relative to the parent of the current sub_path.
```

Comment:
This is *not* equivalent to `line_to(Unit(0), Unit(0))` - subpaths can start anywhere and can have parents. A better narrative description would be something like "draw a line back to the starting position (including any parent) of the current subpath.